gradle/libs.versions.toml

```diff
@@ -1,45 +1,43 @@
 [versions]
-nva = { strictly = '1.41.0' }
```

Comment:
I sorted these keys (Idea: select text, Edit > Sort lines) since I find it difficult to navigate when they are not sorted. 

This makes the review a bit difficult, but I have annotated the changes.